### DISCUSSION TITLE
WIFI大流量，路由器无线重启

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#129] [Bugs/问题反馈]
小米ax3600,515内核，只要WiFi 5g频段，大流量，就会无限重启断网，好像是内存泄漏
网线直接插电脑不会

**Comment by Boos4721** [comment]
<img width="1034" alt="iShot_2022-05-05_23 07 51" src="https://user-images.githubusercontent.com/43845654/166954220-41bc5316-7ccc-4045-b4cb-705d27c4fa3c.png">

**Reply by vonl1** [reply]
![image-20220507093827042](https://user-images.githubusercontent.com/40910092/167233038-f3bdaba1-7eef-4305-97c4-1cc30c7c2735.png)
![image-20220507093809774](https://user-images.githubusercontent.com/40910092/167233057-41b65b44-3394-4330-92f5-30e782929d6a.png)
单开openclash + ax 160mhz wifi + 手机，九分钟不到死了

**Comment by vonl1** [comment]
**上次显示wan6占用cpu多,这次把wan6关掉,还是openclash + ax 160 mhz wifi  + 手机，撑到9分33秒,比上次强了44秒**
![image](https://user-images.githubusercontent.com/40910092/167233514-2615d31d-f65a-4a3f-9bb6-262d7d52eda9.png)

**Reply by YuXilong** [reply]
我也一样 刷了5.15就会不断死机重启 退回5.10了

**Reply by he110777** [reply]
5月6日更新之后我的AX6一直没有重启。

**Reply by he110777** [reply]
ax3600没有uboot还一直在用5.10。

**Reply by Boos4721** [reply]
> ax3600没有uboot还一直在用5.10。

kk

**Reply by vonl1** [reply]
> > ax3600没有uboot还一直在用5.10。
> 
> kk

前两天的包好了,wifi不掉内存了
开了以下程序,测试稳定一天17小时了
- openclash
- adguard home 
- 2.4Gwifi
- 5G 160mhz wifi
![image](https://user-images.githubusercontent.com/40910092/167524965-8e106829-cee1-4f94-8b4f-d100964d4577.png)

**Reply by he110777** [reply]
你有用upnp吗，能正常映射端口吗？
